src/Cardano/Wallet/Api/Types.hs

```diff
@@ -147,15 +149,10 @@ data WalletPutPassphraseData = WalletPutPassphraseData
     } deriving (Eq, Generic, Show)
 
 data CreateTransactionData = CreateTransactionData
-    { _targets :: ![TargetOutput]
+    { _targets :: ![TxOut]
```

Comment:
@KtorZ what do you think about reusing tx output here and a bit bellow how aeson instances are handled

Reply:
I have abandoned this idea as it didn't quite suite my needs - I had to wrap fields of `TxOut` with `ApiT` and it was easier to opt for a separate type.

ps - If number of such types increases, we could use template haskell to generate such types, types - but number of types is low so there is no need for such heavy machinery at this point (and I am not sure what is our opinion about template haskell in general).

For example if we had something like:
```haskell
deriveApiTFor TxOut
```
something like this would create type
```haskell
data TxOut = TxOut
  { address :: ApiT Address
  ,  coin :: ApiT Coin
  }
```

As personally I run only to one such use case so far I didn't want to introduce anything like the solution above - but if there were many such examples we might consider it